modin/pandas/base.py

```diff
@@ -85,6 +85,7 @@
     "name",
     "dtypes",
     "dtype",
+    "groupby",
```

Comment:
`GroupbyObject` now has its own default-to-pandas logic, so we want default behavior for its constructor